### PR TITLE
[docker] fixes #134 - Modify Dockerfile to fix sqlite problem

### LINKED ISSUE #134: Builds in Docker Cloud are failing due to the sqlite package

### PR DESCRIPTION
**Pull request by stdevNorge**
Fixes #134 

Changes:
- Update to golang 1.10
- Install some dependencies to Alpine
- Enable cgo 

Does this change need to mentioned in CHANGELOG.md?
No

### CODE REVIEW

File: hooks/push [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 docker push $DOCKER_REPO
-docker push $DOCKER_REPO:arm32v5
+# docker push $DOCKER_REPO:arm32v5
 docker push $DOCKER_REPO:arm32v6
-docker push $DOCKER_REPO:arm32v7
-docker push $DOCKER_REPO:arm64v8
+# docker push $DOCKER_REPO:arm32v7
```
Comment (olemis) [inline hooks/push]: @stdevNorge in order to enable this images should be built upon `busybox`

File: hooks/push [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+# docker push $DOCKER_REPO:arm64v8
```
Comment (olemis) [inline hooks/push]: This one should work with `alpine` image . What's wrong ?

**Review by olemis — CHANGES_REQUESTED** [review]

File: hooks/push (hunk 2 above)
Comment (olemis) [reply hooks/push]: You are building it and not publishing in the Hub ? Why ?

**Comment by gz-c** [comment]
@olemis is this one good now?

**Review by olemis — CHANGES_REQUESTED** [review]
@gz-c the only thing missing is a comment in README about ARM base images `alpine` for `armv6`, `arm64v8` and `busybox` for `armv5`, `arm7`. @stdevNorge will get this done asap

**Comment by olemis** [comment]
@gz-c ARM builds working as expected . See [tags in unofficial image repository](https://hub.docker.com/r/simelotech/skywire/tags/).

**Comment by olemis** [comment]
@gz-c English and Spanish README are complete . Maybe we'll need an update for other languages as well . Other than this I'd only suggest to wait for the Docker Cloud build triggered by latest commits. If they succeed it's ready to merge .

**Pull request merged**